src/main/java/com/google/sps/data/TextAnalyser.java

```diff
@@ -143,12 +145,10 @@ private ClassifyTextResponse classify() throws IOException {
   }
 
   public Set<String> getEvents() {
-    Set<String> events = new HashSet<String>();
-    String[] allEvents = new String[] {"birthday", "wedding", "baby shower", "love",
-                                    "congratulation", "travel", "good morning",
-                                    "gratitude", "job", "promotion",
-                                    "new", "welcome", "good evening", "good night",
-                                    "holiday"};
+    Set<String> events = new LinkedHashSet<String>();
+    String[] allEvents = new String[] {"birthday", "wedding", "baby shower", "travel", 
```

Comment:
Any reason to don't use an Enum class here?

Reply:
Also Mara's